### PR TITLE
Tickets/dm 46949

### PR DESCRIPTION
**Pull request by jbkalmbach**

### CODE REVIEW

File: pipelines/production/comCamRapidAnalysisPipeline.yaml [hunk 1]
```diff
@@ -30,22 +30,22 @@ tasks:
...
     class: lsst.donut.viz.PlotAOSTask
     config:
-      doRubinTVUpload: false
+      doRubinTVUpload: true
   aggregateDonutStampsTask:
     class: lsst.donut.viz.AggregateDonutStampsTask
   plotDonutTask:
     class: lsst.donut.viz.PlotDonutTask
     config:
-      doRubinTVUpload: false
```
Comment (jmeyers314) [inline pipelines/production/comCamRapidAnalysisPipeline.yaml:48]: Do we have a nonproduction pipeline where this stays False?  Maybe we can have the production pipeline import the nonproduction one and just make this one change?

File: tests/task/test_generateDonutCatalogUtils.py [hunk 2]
```diff
@@ -348,7 +344,66 @@ def testAddVisitInfoToCatTable(self):
...
+            roundTripVisitInfo.boresightRaDec, testExposure.visitInfo.boresightRaDec
+        )
+        if testExposure.visitInfo.boresightAzAlt.isFinite():
+            # Test that they are equal if they are not nan
+            self.assertEqual(
+                roundTripVisitInfo.boresightAzAlt, testExposure.visitInfo.boresightAzAlt
+            )
+        else:
+            # If testExposure has nan value, round trip should as well
+            self.assertFalse(roundTripVisitInfo.boresightAzAlt.isFinite())
```
Comment (gmegh) [inline tests/task/test_generateDonutCatalogUtils.py:385]: remove empty line

File: python/lsst/ts/wep/task/donutStampSelectorTask.py [hunk 3]
```diff
@@ -118,6 +118,14 @@ def run(self, donutStamps):
                         if result.selected[i]
                     ]
                 )
+        for key, val in donutStamps.metadata.items():
+            if key.startswith("BORESIGHT"):
```
Comment (gmegh) [inline python/lsst/ts/wep/task/donutStampSelectorTask.py]: looks like you couuld handle this with only one 'if' and 'or'?

**Review by gmegh — APPROVED** [review]
Left some comments below in addition to josh's suggestion, in case you'd like to consider them. Other than that, it looks good to me.

File: pipelines/production/comCamRapidAnalysisPipeline.yaml (hunk 1 above)
Comment (jbkalmbach) [reply pipelines/production/comCamRapidAnalysisPipeline.yaml:48]: Merlin suggested some pipeline changes that we should implement like having the ComCam and ComCamSim pipeline import the same basic ingredients. I can add including a nonproduction, testing pipeline to the list for that.

Comment (jbkalmbach) [reply pipelines/production/comCamRapidAnalysisPipeline.yaml:48]: https://rubinobs.atlassian.net/browse/DM-47109

File: tests/task/test_generateDonutCatalogUtils.py (hunk 2 above)
Comment (jbkalmbach) [reply tests/task/test_generateDonutCatalogUtils.py:385]: I don't see an empty line here. Not sure what you're referring to.

File: python/lsst/ts/wep/task/donutStampSelectorTask.py (hunk 3 above)
Comment (jbkalmbach) [reply python/lsst/ts/wep/task/donutStampSelectorTask.py]: Updated

**Pull request merged**